FirebaseAuth/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- [fixed] Fix the parameter from "languageCode" to "lang" in ActionCodeURL
```

Comment:
Thanks. Please add the issue number that is being fixed, similar to the other entries in this file - #14664.

Reply:
Oops, my bad. Thanks for pointing it out, Paul :)